wkcuber/wkcuber/download.py

```diff
@@ -0,0 +1,130 @@
+"""This module takes care of downsampling WEBKNOSSOS datasets."""
+
+from pathlib import Path
+from typing import Annotated, List, Optional
+
+import typer
+from rich import print as rprint
+
+from webknossos import BoundingBox, Dataset, Mag, webknossos_context
+from webknossos.client._defaults import DEFAULT_WEBKNOSSOS_URL
+from wkcuber.utils import parse_bbox, parse_mag
+
+app = typer.Typer(invoke_without_command=True)
+
+
+@app.callback()
+def main(
+    *,
+    path: Annotated[
+        Path,
+        typer.Argument(
+            show_default=False,
+            help="Path to save your WEBKNOSSOS dataset.",
+        ),
+    ],
+    exist_ok: Annotated[
+        bool,
+        typer.Option(
+            help="Is it alright to overwrite existing data on download path.",
```

Comment:
What is a download path? This option refers to the output or target path on disk. Perhaps rephrase as "Allow overwriting of dataset if it already exists on disk"